### DISCUSSION TITLE
BAG v2 script laadt statuskolommen niet goed in

### DISCUSSION THREAD

**Discussion by simonewesselius** [nlextract/NLExtract#396] [Q&A] [answered]
In het script BAG_bagv2-dump-db-schema-v1.sql lijkt het niet goed te gaan bij het creëren van tabellen met daarin een status kolom. Zie onderstaand voorbeeld van bagactueel.pand waarin het datatype van pandstatus niet goed gedefinieerd wordt.

![image](https://github.com/user-attachments/assets/464d1b09-03ea-4041-b5f0-12d0106c8a26)

**Comment by justb4** [comment, marked as answer]
Dit is een vraag voor support@geotoko.nl . `bagv2-dump-db-schema-v1.sql` is bedoeld om het specifieke [schema te beschrijven](https://geocatalogus.nl/dataset/bag-extract-database-dumps/resource/1a1b09fe-0c4a-407e-9417-84f0e19a09ae), zoals in het geotoko.nl product [BAG Extract PostGIS Dumps](https://geocatalogus.nl/dataset/bag-extract-database-dumps), niet zozeer om in te lezen. Kun je de vraag met meer context: welke omgeving, welke handelingen, welke bedoeling in de mail naar support@geotoko.nl beschrijven?

Ik vond overigens geen probleem om het [hele schema bagv2-dump-db-schema-v1.sql](https://geocatalogus.nl/dataset/bag-extract-database-dumps/resource/1a1b09fe-0c4a-407e-9417-84f0e19a09ae) in een lege PostGIS DB uit te voeren...

**Reply by simonewesselius** [reply]
Bedankt. Het inladen is gelukt en ik begreep dat de SQL niet gebruikt was om in te laden. Ik kan nu het issue met het datatype niet reproduceren. Als dit opnieuw voorkomt, dan zal ik dit aangeven. Voor nu is het in ieder geval opgelost.
